qemu/tests/cfg/multi_nics_stress.cfg

```diff
@@ -6,10 +6,10 @@
     image_snapshot = yes
     hostpassword = redhat
     # nic1 is for control, nic2 is for data connection
-    vms = "vm1 vm2 vm3 vm4 vm5"
+    vms += " vm2 vm3 vm4 vm5"
     netperf_client = ${main_vm}
     netperf_server = "vm2 vm3 vm4 vm5"
-    nics_vm1 = 'nic1 nic2 nic3 nic4'
+    nics_avocado-vt-vm1 = 'nic1 nic2 nic3 nic4'
```

Comment:
It is OK to me.
But I think it may be better to use same naming pattern for all vms.

Reply:
@Zhengtong I will open another bug to follow dhclient other 3 nics problem.